src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -87,28 +89,57 @@ if (__DEV__) {
       // We were unable to guess which prop the user intended.
       // It is likely that the user was just blindly spreading/forwarding props
       // Components should be careful to only render valid props/attributes.
-      warning(
-        false,
-        'Unknown prop `%s` on <%s> tag. Remove this prop from the element. ' +
-        'For details, see https://fb.me/react-unknown-prop%s',
-        name,
-        tagName,
-        ReactComponentTreeDevtool.getStackAddendumByID(debugID)
-      );
+      unknownProp = name;
```

Comment:
The calling function already knows the name of the unknown prop—it passes it as an argument. Let’s just return `true` if there is no need for additional warning, and `false` if we still need to warn. We can rename `handleProperty` to `checkKnownProperty`.